driver/random.hpp

```diff
@@ -1,32 +1,46 @@
 #ifndef GUARD_RANDOM_GEN_
 #define GUARD_RANDOM_GEN_
 
+#include <miopen/env.hpp>
+#include <iostream>
+#include <cstdlib>
 #include <random>
 
 std::minstd_rand& get_minstd_gen()
 {
-    static thread_local std::minstd_rand minstd_gen(std::random_device{}());
+    static thread_local std::minstd_rand minstd_gen{[]() {
+        auto external_seed = miopen::EnvvarValue("MIOPEN_DRIVER_PRNG_SEED", 100500);
+
+        auto seed = external_seed == 0
+                        ? std::random_device{}()
+                        : static_cast<std::random_device::result_type>(external_seed);
+        std::cout << "Random seed: " << seed << "\n";
+        return seed;
+    }()};
+
     return minstd_gen;
 }
 
-template <typename T>
-inline T FRAND()
+bool use_legacy_prng()
 {
-    double d = std::generate_canonical<double, 1>(get_minstd_gen());
-    return static_cast<T>(d);
+    static bool legacy_prng = miopen::IsEnvvarValueEnabled("MIOPEN_USE_LEGACY_PRNG");
+    return legacy_prng;
```

Comment:
```suggestion
    return miopen::IsEnabled("MIOPEN_DRIVER_PRNG_LEGACY"); // Allows reusing of old verification cache.
```
Let's unify env var name.